### PR TITLE
New template - File indexing

### PR DESCRIPTION
**Pull request by divyaswarnkar**

### CODE REVIEW

**Comment by divyaswarnkar** [comment]
@Elaina-Lee for review

File: ingest-index-ai-azure-files-schedule-rag/manifest.json [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    "images": {
+        "light": "workflow-light",
+        "dark": "workflow-dark"
+    },
+    "parameters": [
+        {
+            "name": "SearchIndex_#workflowname#",
+            "displayName": "AISearch index name",
+            "type": "String",
+            "description": "Provide the Azure AI search index name",
```
Comment (Elaina-Lee) [inline ingest-index-ai-azure-files-schedule-rag/manifest.json:37]: Nit: Could we add the '.' to the end of the description to align with the other parameters?
`Provide the Azure AI search index name.`

File: ingest-index-ai-azure-files-schedule-rag/manifest.json [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            "displayName": "Azure OpenAI text embedding deployment model name",
+            "type": "String",
+            "description": "Provide the Azure OpenAI embedding deployment model name to generate embeddings.",
+            "required": true
+        },
+        {
+            "name": "AzureFileShareName_#workflowname#",
+            "displayName": "Azure File share name",
+            "type": "String",
+            "description": "Provide the Azure File share name",
```
Comment (Elaina-Lee) [inline ingest-index-ai-azure-files-schedule-rag/manifest.json:51]: Nit: Could we add the '.' to the end of the description to align with the other parameters?

File: ingest-index-ai-azure-files-schedule-rag/manifest.json [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+            "required": true
+        },
+        {
+            "name": "StartTime_#workflowname#",
+            "displayName": "Start Time for Ingestion (Recurrence Trigger)",
+            "type": "String",
+            "description": "Provider the start time for ingestion. Use the format - YYYY-MM-DDThh:mm:ssZ. For example:  September 18, 2017 at 2:00 PM would be 2017-09-18T14:00:00Z",
```
Comment (Elaina-Lee) [inline ingest-index-ai-azure-files-schedule-rag/manifest.json:58]: Nit: Could we add the '.' to the end of the description to align with the other parameters?

**Review by Elaina-Lee — APPROVED** [review]

**Comment by divyaswarnkar** [comment]
@Elaina-Lee @preetriti1 please review. All my changes are in

File: ingest-index-ai-one-drive-schedule-rag/manifest.json [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    "images": {
+        "light": "workflow-light",
+        "dark": "workflow-dark"
+    },
+    "parameters": [
+        {
+            "name": "SearchIndex_#workflowname#",
+            "displayName": "Azure AI Search index name",
+            "type": "String",
+            "description": "Provide the Azure AI Search index name",
```
Comment (Elaina-Lee) [inline ingest-index-ai-one-drive-schedule-rag/manifest.json:37]: Nit: Could we end those with '.'?

File: ingest-index-ai-one-drive-schedule-rag/manifest.json [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+            "displayName": "One Drive Folder Name",
+            "type": "String",
+            "description": "Provide the OneDrive folder name for your files. It should be a folder in root folder.",
+            "required": true
+        },
+        {
+            "name": "StartTime_#workflowname#",
+            "displayName": "Start Time for Ingestion (Recurrence Trigger)",
+            "type": "String",
+            "description": "Provide the start time for ingestion. Use the format - YYYY-MM-DDThh:mm:ssZ. For example:  September 18, 2017 at 2:00 PM would be 2017-09-18T14:00:00Z",
```
Comment (Elaina-Lee) [inline ingest-index-ai-one-drive-schedule-rag/manifest.json:58]: Nit: Could we end those with '.'?

**Review by Elaina-Lee — APPROVED** [review]

**Pull request merged**